packages/hash/frontend/src/pages/[account-slug]/types/entity-type/use-entity-type.tsx

```diff
@@ -63,13 +64,14 @@ export const useEntityTypeValue = (
           : [];
 
         /** @todo - pick the latest version? */
-        const relevantEntityType = relevantEntityTypes
+        const relevantEntityType = relevantEntityTypes.length
           ? relevantEntityTypes[0]!.inner
           : null;
 
         await loadTypeSystem();
 
         if (!cancelled) {
+          setLoading(false);
```

Comment:
This also needs to be set to true when the request begins, for when `entityTypeBaseUri` changes

Reply:
Done in abff84d7f17d08fd0cd9136a28ce010b988d5b97.